src/market/market.py

```diff
@@ -181,8 +214,36 @@ def place_order(  # pylint: disable=too-many-arguments
                 open_order_accounts[0].address if open_order_accounts else new_open_orders_account.public_key(),
             )
         )
+
+        if should_wrap_sol:
+            transaction.add(
+                close_account(
+                    CloseAccountParams(
+                        account=wrapped_sol_account.public_key(),
+                        owner=owner.public_key(),
+                        dest=owner.public_key(),
+                    )
+                )
+            )
+
         return self._send_transaction(transaction, *signers)
 
+    @staticmethod
+    def get_lamport_need_for_sol_wrapping(
+        price: int, size: int, side: Side, open_orders_accounts: List[OpenOrdersAccount]
```

Comment:
Can't we just pass `open_orders_accounts[0]` here instead of the entire list?

Reply:
If we do it, we'll need to check outside if the list is empty and make the argument optional. It seems more work to me.

Reply:
The same problem exists here right? You should be checking if the list empty? Not sure why you would make this argument optional.